scripts/dev/templates/operator/Dockerfile.builder

```diff
@@ -0,0 +1,20 @@
+ARG builder_image
+FROM ${builder_image} AS builder
+
+RUN mkdir /workspace
+WORKDIR /workspace
+
+COPY go.mod go.sum ./
+COPY cmd/manager/main.go cmd/manager/main.go
+COPY api/ api/
+COPY controllers/ controllers/
+COPY pkg/ pkg/
+COPY build/bin/ build/bin/
+
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager cmd/manager/main.go
+
+FROM busybox
```

Comment:
`scratch` would work better here.

Reply:
I wanted to preserve the ability to run the context image, e.g. for investigating what's inside. If we can do this using scratch (which I doubt) then I could change it. Otherwise I would leave it as busybox is ~1MB.

Reply:
Also this is something that was before.